internal/brokers/broker.go

```diff
@@ -305,3 +287,76 @@ func (b Broker) validateUILayout(sessionID string, layout map[string]string) (r
 func (b Broker) parseSessionID(sessionID string) string {
 	return strings.TrimPrefix(sessionID, fmt.Sprintf("%s-", b.ID))
 }
+
+// validateAndMarshalUser checks if the specified rawMsg is a valid userinfo and marshals it as a JSON string.
+func (b Broker) validateAndMarshalUser(rawMsg json.RawMessage) (data string, err error) {
+	defer decorate.OnError(&err, "invalid user information")
```

Comment:
or (instead of my next comment because I can predict the future :p): print the rawMsg here, so that any error will see the raw content.

Reply:
Isn't this prone to exposing "too much" information? Like information leaks and so on, as the broker could, technically, return anything?

Reply:
Well, the auth was successful, so it’s user information. There is no password in it, so I think this is fine.